test/phoenix_live_view/html_formatter_test.exs

```diff
@@ -487,6 +493,52 @@ if Version.match?(System.version(), ">= 1.13.0") do
       )
     end
 
+    test "text between inline elements" do
+      assert_formatter_doesnt_change(
+        """
+        <span><%= @user_a %></span>
+        X
+        <span><%= @user_b %></span>
+        """,
+        line_length: 27
+      )
+
+      assert_formatter_doesnt_change(
```

Comment:
That's tricky. I agree it would look better formatted as `<span><%= @user_a %></span> X <span><%= @user_b %></span>` but it does interfere with other rules, so the idea here is to keep the text unchanged if it's surrounded by inline elements + line break.